stories/screen.js

```diff
@@ -3,12 +3,20 @@ import { storiesOf } from '@storybook/react';
 import WelcomeView from '../src/view/welcome';
 import HomeView from '../src/view/home';
 import { Store } from '../src/store';
+import PaymentRequest from '../src/view/payment-request';
 
 const store = new Store();
 
 storiesOf('Screens', module)
   .add('Welcome', () => <WelcomeView />)
-  .add('Home', () => <HomeView store={store} />);
+  .add('Home', () => <HomeView store={store} />)
+  .add('Payment Request', () => (
+    <PaymentRequest
+      amount="456,780"
+      invoice="lnbc4567800n1pdvqx48pp5eng6uyqnkdlx93m2598ug93qtuls8gapygxznshzd56h7n5cxs0sdp9gehhygr5dpjjqmr0wejjqmmxyp3xjarrda5kucqzysmhyrleqpt3yqf5nctzsr3hvrv9vhhnawazkwyzu8t4mf85tllsyjsf8hgu5nt6dj3jaljjgmt999xnlsweqvatypzlu34nhpjlxf59qp4dn2pv"
+      message="For the love of bitcoin"
```

Comment:
Injection individual attributes won't work unfortunately. You need to pass by reference not by value. Otherwise the mobx observer won't be notified of state updates. See: https://mobx.js.org/refguide/observer-component.html#gotcha-dereference-values-inside-your-components

Ideally we'd pass the whole mobx store into each view like so `<HomeView store={store} />`. That way we have a consistent way to render views from a single source of truth :)